task-manager.dart

```diff
@@ -1,53 +1,67 @@
+enum Status { completed, pending }
+
 class Task {
+  // class properties
+  int _id;

```

Comment:
Id wasn't part of the properties, but I think it would the system ability to get the tasks